src/pages/index.js

```diff
@@ -17,47 +42,111 @@ popupImage.setEventListeners();
 const openPopupImage = (data) => {
   popupImage.open(data);
 };
-const renderCard = (card) => {
-  cardRenderer.addItem(createCard(card));
+
+const deletePopup = new PopupWithForm(constants.deletePopup, (data) => {
+  api
+    .deleteCard(data._id, data)
+    .then(() => {
+      deletePopup.submitButton.textContent = "Delete Card";
+      cardArray = document.querySelectorAll(".card");
+      cardArray.forEach((card) => {
+        if (card.id === data._id) {
+          card.remove();
+        }
+      });
+      deletePopup.close();
+    })
+    .catch((error) => {
+      alert(`Request to server failed. ${error}`);
+    });
+});
+deletePopup.setEventListeners();
+
+const openDeletePopup = (data) => {
+  deletePopup.open();
+  constants.deleteID.value = data._id;
 };
-const cardRenderer = new Section(
-  { items: constants.initialCards, renderer: renderCard },
-  constants.cardList
-);
 
-cardRenderer.renderItems();
+function sendLike(data) {
+  if (data.isLiked) {
+    api
+      .removeLike(data._id, data)
+      .then((data.isLiked = false))
+      .then(alert(`Like for "${data.name}" removed`));
```

Comment:
`      .then(alert(`Like for "${data.name}" removed`));
`

Please, delete it . You don't need it